### PR TITLE
[REF] global refactoring: better message output and use real file and line number in non-py files

### PR DESCRIPTION
**Pull request by moylop260**
Change from
`pylint_odoo/test_repo/broken_module/`**`__init__`:1:**
[W7910(wrong-tabs-instead-of-spaces), ]  **model_view_odoo.xml:23** Use wrong tabs indentation instead of four spaces`

to `pylint_odoo/test_repo/broken_module/`**model_view_odoo.xml:23:**
[W7910(wrong-tabs-instead-of-spaces), ] **(nothing)** Use wrong tabs indentation instead of four spaces`

### CODE REVIEW

File: pylint_odoo/checkers/modules_odoo.py [hunk 1]
```diff
@@ -14,12 +14,12 @@
 
     # Visit odoo module with settings.BASE_OMODULE_ID
     'C%d02' % settings.BASE_OMODULE_ID: (
-        'Missing ./README.rst file. Template here: %s',
+        'README.rst missing file. Template here: %s',
```
Comment (pedrobaeza) [inline pylint_odoo/checkers/modules_odoo.py]: This text is correct, so no need to change it

**Comment by moylop260** [comment]
This PR use a better messages in the log
See example: https://travis-ci.org/OCA/pylint-odoo/jobs/154304878#L545

Your feedback is welcome here

**Comment by oscarolar** [comment]
nice 👍

**Comment by alan196** [comment]
:+1:

**Comment by moylop260** [comment]
@pedrobaeza 
Could we merge it?

**Comment by pedrobaeza** [comment]
:+1:

**Comment by moylop260** [comment]
Thanks!

**Pull request merged**